db/primary_migrate/20241015154109_create_socure_reason_codes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateSocureReasonCodes < ActiveRecord::Migration[7.1]
+  def change
+    create_table :socure_reason_codes do |t|
+      t.string :code, comment: 'sensitive=false'
+      t.string :group, comment: 'sensitive=false'
+      t.text :description, comment: 'sensitive=false'
+      t.datetime :added_at, comment: 'sensitive=false'
+      t.datetime :deactivated_at, comment: 'sensitive=false'
+
+      t.timestamps comment: 'sensitive=false'
```

Comment:
Should we just rely on the stock `created_at` this gives us vs. having an `added_at` column?

Reply:
I don't like relying on `created_at` for these sorts of things. Funny stuff can happen when records get copied and things like that.